src/app/app-layout/navbar/navbar.component.html

```diff
@@ -1,57 +1,87 @@
-<div class="bg-white z-0 flex w-screen flex-col lg:flex-row justify-start lg:justify-center transition-all" [class.h-screen]="(isDropdownOpen$ | async)">
-    <div class=" flex w-full justify-start h-24 sticky overflow-x-hidden max-w-6xl px-4 lg:px-0"
```

Comment:
`justify-start` is the default value for `display: flex`, therefore it's redundant.